css/main.css

```diff
@@ -591,3 +559,54 @@ li.navbar-link:hover a, li.navbar-sublink:hover {
 .navbar-link, .navbar-sublink {
     text-align: center;
 }
+
+.gallery {
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    padding: 4vw;
+    flex-wrap: wrap;
+    width: 90vw;
+}
+
+.staff {
+    width: 100%;
+}
+
+.staff-container {
+    margin: 2vh;
+}
+
+.staff-container div.staff-image {
+    width: 22vh;
+    height: 22vh;
+    overflow: hidden;
+    background-size: cover;
+    padding: 0;
+    border-radius: 50%;
+}
+
+.staff-container:hover div.staff-image {
+    box-shadow:0 4px 12px 0 var(--color-translucent), 0 6px 20px 0 var(--color-translucent);
```

Comment:
What does this do?

Reply:
Adds a shadow when you hover over someone's photo / text.
Like this: 
![image](https://user-images.githubusercontent.com/13492296/58417040-380d5a80-80a1-11e9-9e9a-ee3002dec84a.png)